downloads/swan-lake-release-notes/swan-lake-2201.11.0/RELEASE_NOTE.md

```diff
@@ -169,6 +169,10 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 11 (2201.11.0)
 
 ### Improvements
 
+- Replaced `Package.md` with `README.md` as the primary documentation for packages, ensuring a unified documentation format across Ballerina Central and version control platforms. The default `README.md` can be overridden by specifying a preferred file using the `readme` field in the [package] table of the `Ballerina.toml`.
+- Introduced the `[[package.modules]]` array in the Ballerina.toml to define module-specific metadata, such as export and readme.
```

Comment:
```suggestion
- Introduced configuration support to specify metadata, including enabling export and defining a custom readme file name for a specific module.
```

Reply:
Shall we add an example, rather than mentioning the `[[package.modules]]` array in the explanation?

Reply:
Added with a61e7a2db16807cc17377b542957940bbae66d9c